src/main/java/martin/task/Event.java

```diff
@@ -0,0 +1,17 @@
+package martin.task;
+
+public class Event extends Task {
+    protected String from;
+    protected String to;
+
+    public Event(String description, String from, String to) {
+        super(description);
+        this.from = from;
+        this.to = to;
+    }
+
+    @Override
+    public String toString() {
+        return "[E][" + (isDone ? "X" : " ") + "] " + description + " (from: " + from + " to: " + to + ")";
```

Comment:
Consider using String.format here, it makes the string more readable and easier to edit if required in the future.